### PR TITLE
add event when a variable is updated

### PR DESCRIPTION
**Pull request by noodom**
add an event to notify that a variable has been updated, for use by users scenarios or plugins

### CODE REVIEW

File: core/class/dataStore.class.php [hunk 1]
```diff
@@ -107,6 +107,9 @@ public function save() {
 	}
 
 	public function postSave() {
+		if ($this->getType() == 'scenario') {
```
Comment (ColonelMoutarde) [inline core/class/dataStore.class.php:110]: use strict comparison "==="

**Comment by zoic21** [comment]
Hello,
I don't understand the purpose of this PR...

**Comment by sagitaz** [comment]
Voici une demande faite sur jeemate.

**Comment by Mips2648** [comment]
Hi,

Tell @pyreneum to not use a variable for that but a virtual info instead and problem solved.

**Comment by Salvialf** [comment]
https://github.com/JeeMateTeam/JeeMate-Project/issues/220

**Pull request closed without merging**